bigbluebutton-html5/imports/ui/components/timer/panel/component.tsx

```diff
@@ -219,19 +271,39 @@ const TimerPanel: React.FC<TimerPanelProps> = ({
     syncTimeWithBackend(h, m, s);
   }, [running, hours, minutes, seconds]);
 
-  const handleIncrement = useCallback(() => {
-    let incrementAmount = 1;
-    if (focusedUnit === 'minutes') incrementAmount = 60;
-    if (focusedUnit === 'hours') incrementAmount = 3600;
-    changeTime(incrementAmount);
-  }, [focusedUnit, changeTime]);
+  const setAbsoluteTime = useCallback((totalSeconds: number) => {
+    if (running) return;
+    const h = Math.floor(totalSeconds / 3600);
+    const m = Math.floor((totalSeconds % 3600) / 60);
+    const s = totalSeconds % 60;
+    syncTimeWithBackend(h, m, s);
+  }, [running, syncTimeWithBackend]);
 
-  const handleDecrement = useCallback(() => {
-    let decrementAmount = -1;
-    if (focusedUnit === 'minutes') decrementAmount = -60;
-    if (focusedUnit === 'hours') decrementAmount = -3600;
-    changeTime(decrementAmount);
-  }, [focusedUnit, changeTime]);
+  const formatPresetLabel = useCallback((totalSeconds: number) => {
+    const hours = totalSeconds / 3600;
+    if (Number.isInteger(hours) && hours >= 1) {
+      return `${hours}${intl.formatMessage(intlMessages.hoursAbbr)}`;
```

Comment:
Please include hours/minutes variables in the intl message instead of concatenating it, as concatenation may cause issues in RTL languages (see https://github.com/bigbluebutton/bigbluebutton/pull/21660)